packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -1621,6 +1621,7 @@ abstract class FlutterCommand extends Command<void> {
       commandPath: commandPath,
       result: commandResult.toString(),
       maxRss: maxRss,
+      commandHasTerminal: globals.stdio.hasTerminal,
```

Comment:
with package:unified_analytics, are we only tracking this on command results, or will we have that data with every event? I'm still unclear on https://github.com/dart-lang/tools/issues/197#issuecomment-1802982835

Reply:
That's a good question, i was actually doing some digging and it doesn't seem like that boolean is getting sent with each event, only for the events that indicate which event was run.

It's not set at the session level

Reply:
Ahh, but was this the only context we were providing the custom dimension with the legacy analytics? https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/runner/flutter_command.dart#L1740

Reply:
I believe so, I went through the code base to find any instances of the `commandHasTerminal` custom dimension. I also checked in the database for GA3 and didn't see it being sent with every event.

I believe the only two dimensions we send with every event is the channel and host os. Below is a snippet of the first record in the table

<img width="307" alt="image" src="https://github.com/flutter/flutter/assets/42216813/149fe0f8-5c2b-44f8-9c34-bf87fa322f0c">

Reply:
SGTM